File: 231229/배수의 개수/number-of-multipliers.py
cnt_3 = 0
cnt_5 = 0
for _ in range(10):
    num = int(input())
    if num % 3 == 0:
        cnt_3 += 1
    if num % 5 == 0:
        cnt_5 += 1

print(cnt_3, end=' ')
print(cnt_5)

File: 240530/배수의 개수/number-of-multipliers.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int cnt3 = 0;
        int cnt5 = 0;
        for(int i = 1; i <= 10; i++) {
            int a;
            a = sc.nextInt();
            if(a%3 == 0) {
                cnt3++;
            }
            if(a%5==0) {
                cnt5++;
            }
        }
        System.out.print(cnt3 + " " + cnt5);
    }
}